ls-refs.c

```diff
@@ -90,6 +94,16 @@ static int send_ref(const char *refname, const struct object_id *oid,
 	if (!ref_match(&data->prefixes, refname_nons))
 		return 0;
 
+	if (!strcmp(refname_nons, "HEAD")) {
```

Comment:
use strncmp its safer

Reply:
thanks, will do it.

Reply:
Umm. Why would it be safer? Both `refname_nons` and `HEAD` are NUL-terminated. In this instance, `strcmp()` is the right thing to call.